.travis.yml

```diff
@@ -7,15 +7,39 @@ sudo: required
 before_install:
   - if [[ "$INSTALL_ARM_DEPS" == "yes" ]]; then tools/apt-get-install-arm.sh; fi
   - if [[ "$INSTALL_NUTTX_DEPS" == "yes" ]]; then tools/apt-get-install-nuttx.sh; fi
+  - if [[ "$INSTALL_TRAVIS_I686_DEPS" == "yes" ]]; then tools/apt-get-install-travis-i686.sh; fi
   - tools/apt-get-install-deps.sh
 
 install:
 
 script: "tools/precommit.py $OPTS"
 
 env:
-  - OPTS="--test=host --buildtype=release"
-  - OPTS="--test=host --buildtype=debug"
+  - OPTS="--test=host"
   - OPTS="--test=rpi2" INSTALL_ARM_DEPS=yes
   - OPTS="--test=nuttx" INSTALL_NUTTX_DEPS=yes
   - OPTS="--test=misc"
+
+matrix:
+  include:
+    - compiler: gcc-4.9
```

Comment:
@bzsolt actually gcc-4.8 is used for all compilation, except arm-linux-gnueabi, where gcc 4.7 is default on trusty :) In this case JerryScript does not compile. Compiler version is really important. Having linux test on 4.9, arm builds on 4.8 could lead to unexpected, hard to find problems.